linux_os/guide/system/accounts/accounts-restrictions/root_logins/root_path_default/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8
+prodtype: fedora,ol7,ol8,rhel7,rhel8,sle15
```

Comment:
You didn't add a `cis` reference for this rule. Is that intentional?

Reply:
No it is not. I'll add it later today. Thanks for pointing it out.

Reply:
After checking it, I removed the prodtype 'sle15' for now and commented out the reference in the profile.

Because I'm not sure that this rule can return the correct result for the use case ("Ensure root PATH Integrity").